app/templates/components/preprint-form-authors.hbs

```diff
@@ -113,14 +113,28 @@
                 {{#sortable-group tagName="tbody" onChange="reorderItems" as |group|}}
                     {{#each contributors as |contrib|}}
                         {{#sortable-item tagName="tr" model=contrib class="contributor-row" group=group spacing=1 handle=".handle" id=contrib.id}}
-                            <td class="text-nowrap">
+                            <td class="text-nowrap author-cols">
                                 <div class="form-group drag-drop-contrib">
                                     <span class="fa fa-bars sortable-bars handle small"></span>
                                     <img class="m-l-xs" src={{contrib.users.profileImage}} height=30 width=30>
+                                    <span class="visible-xs-inline"> {{!CONTRIBUTOR NAME - XS Screen}}
+                                        {{#if contrib.unregisteredContributor}}
+                                            {{contrib.unregisteredContributor}}
+                                        {{else}}
+                                            <a href={{contrib.users.links.html}}> {{contrib.users.fullName}} </a>
+                                        {{/if}}
+                                        <span class="pull-right remove-contributor-padding-xs">
+                                            {{#if (and (permissionToRemoveContributor contrib currentUser isAdmin node) (conditionsForContribRemoval contrib contributors authorModification))}}
+                                                <button class="remove-contributor-xs" {{action 'removeContributor' contrib}} aria-label="Remove author from authors list">
```

Comment:
Can we translate the aria-label text as well? Otherwise, this is a hard-coded english string of user facing language, in a page intended for translation.